src/lib/index.js

```diff
@@ -16,23 +17,43 @@ const env = process.env.NODE_ENV || PRODUCTION;
  */
 const getDuration = (now, created) => now - created;
 
+/**
+ * Checks if current environment matches production.
+ * @param {String} environment - The current environment.
+ * @return {Boolean}
+ */
+const isProduction = (environment = env) => environment === PRODUCTION;
+
 module.exports = {
     ExecutionContextResource,
     env,
+    isProduction,
 
     /**
-     * Checks if current environment matches production.
-     * @param {String} environment - The current environment.
+     * Checks if a given value is undefined.
+     * @param {*} thing that thing to check.
      * @return {Boolean}
      */
-    isProduction: (environment = env) => environment === PRODUCTION,
+    isUndefined: (thing) => [null, undefined].includes(thing),
 
     /**
-     * Checks if a given value is undefined.
-     * @param {String} thing that thing to check.
+     * Handles execution context error, throws when none production.
+     * @param {String} code - The error code to log.
+     */
+    handleError: (code) => {
+        if (!isProduction()) {
+            throw code;
+        }
+
+        console.error(code); // eslint-disable-line no-console
```

Comment:
you want to console log this?

Reply:
I dont wanna throw up production

Reply:
Ill keep this convention to enhance the error handling.